### DISCUSSION TITLE
How does podman work with tunnels

### DISCUSSION THREAD

**Discussion by shakeabi** [containers/podman#24520] [Q&A]
Hey everyone. I'm trying to understand how `podman` works when accessing through tunnels. (all of the below podman usages are rootless)
Here's what I've setup currently. I've a VS code tunnel running in my home server (say `home-server`) and connect to it from `vscode.dev` (using MS dev tunnels).

Now accessing the terminal in vscode shows me this 
```
user@home-server:~$ 
```
However, doing `podman ps` here shows nothing although I've about 10 containers running.
```
user@home-server:~/tmp$ podman ps
CONTAINER ID  IMAGE       COMMAND     CREATED     STATUS      PORTS       NAMES
user@home-server:~/tmp$ 
```

Having said that, when I ssh from here to my same account (`ssh user@home-server`) and then do a `podman ps`, it lists all the working containers.

```
user@home-server:~/tmp$ ssh shake@homelab
user@homelab's password: 
user@home-server:~$ podman ps
CONTAINER ID  IMAGE                                        COMMAND               CREATED       STATUS                 PORTS                     NAMES
539252c44171  docker.io/actualbudget/actual-server:latest  node app.js           10 hours ago  Up 10 hours (healthy)  0.0.0.0:5006->5006/tcp    actual-budget
...
```

Can someone help me understand the above behavior? Thanks in advance!

**Comment by Anachron** [comment]
Maybe check if your `env` is different?

Run `env` on both connections and compare them.
